components/Bag.jsx

```diff
@@ -11,12 +16,16 @@ const Bag = ({ name, image, isVegan, variants }) => {
         height={375}
         alt={`Prepackaged sweet mix: ${name}`}
       />
-      <span>{name}</span>
-      <span>Ab: {variants[1].price}€</span>
-      <span>
-        {((variants[1].price / variants[1].weight) * 100).toFixed(2)}€/100g
-      </span>
-      <checkbox checked={isVegan} />
+      <p>
+        {name} {isVegan ? "🌱" : null}
+      </p>
+      <p>Ab: {numberFormat.format(variants[1].price)}</p>
+      <p>
+        {numberFormat.format(
+          ((variants[1].price / variants[1].weight) * 100).toFixed(2)
+        )}
```

Comment:
This is okay. You could make it even clearer if you do the calculation above the return, but I find it perfectly ok.

Reply:
And you should be sure that `weight` is always > 0 or your code will er̶̙̅r̶͝or̷̺͡ a̶̡ͣn̸̬ͣd̴͂ t̸̗ͬh̸︡ḛ̶͝ u̵̹͛n̴ͥ͜i̸͉͘v̵᷂͛e̴͕̍r̸͉᷀s̸̱̔e̵̗︠ ẉ̴̆ị̸᷀ľ̵᷿ḷ̷̀ ç̵̼̯︠̇̇̉ͧ͡o̷̯͉ͫ᷃ͫ̃᷉l̴͈̦̩͓᷁͢͜l̵̖̻̀͢á̴᷿ͧ᷀̕p̷̘͔̼̭ͭ͊̐︢̚͘ŝ̷̡͎̪̦̮̤͉͇̟̫̲᷾̇᷁︡̑᷀̿᷃᷉͘͟ẻ̷̲ͭ̄ͬ.